### DISCUSSION TITLE
selector after and before

### DISCUSSION THREAD

**Discussion by ildevelop** [chakra-ui/chakra-ui#3537] [Q&A] [answered]
hey all ✌️ 
i start using [Chakra Tabs ](https://chakra-ui.com/docs/disclosure/tabs#creating-custom-tab-components) and i also need overwrite `_selected ` ::after border bottom. how can use CSS selector :**:after** here? 
thanks

**Comment by malashevskyi** [comment, marked as answer]
Hi, see this [example](https://codesandbox.io/s/chakra-ui-tabs-after-on-selected-tab-d558d)
